src/libraries/Common/tests/System/Net/Http/GenericLoopbackServer.cs

```diff
@@ -34,7 +34,7 @@ public Task CreateClientAndServerAsync(Func<Uri, Task> clientFunc, Func<GenericL
                 Task serverTask = serverFunc(server);
 
                 await new Task[] { clientTask, serverTask }.WhenAllOrAnyFailed().ConfigureAwait(false);
-            }, options: options).TimeoutAfter(millisecondsTimeout);
+            }, options: options).WaitAsync(TimeSpan.FromMilliseconds(millisecondsTimeout));
```

Comment:
Wouldn't it make sense to also add a `WaitAsync(int millisecondsTimeout)` overload?

Reply:
We can discuss it at the API review.  We didn't add them to ConfigureAwait, so I didn't add them here.  There's also been concern in the past about the decision to use ints for timeouts.  I don't have a strong opinion about adding them.

Reply:
> _There's also been concern in the past about the decision to use ints for timeouts_

I couldn't find info on this, but it got me curious. For my own education, would you mind sharing what the concern is?

Reply:
> what the concern is?

Primarily that it doesn't convey the unit (seconds, milliseconds, etc.)  Secondarily that it's not strongly-typed to the time domain.